### DISCUSSION TITLE
Workflow not triggered when creating tag

### DISCUSSION THREAD

**Discussion by gopac25** [community/community#64491] [Actions]
### Select Topic Area

Bug

### Body

I want my workflow to trigger when I push the tag to github. 

on: 
  push:
    tags:
    - '**'
 
But workflow is not trigger. If I create new release from GH console, job is triggering without any issues. I am not sure what I am missing. If we push the new tag, job should trigger. Any thoughts?

**Comment by Jaber-Saed** [comment]
It sounds like you have set up the GitHub Actions workflow to trigger on tag pushes, but you're experiencing an issue where the workflow is not being triggered when you push a new tag. Let's go through some troubleshooting steps to identify the possible reasons:

1. **Tag Format:**
   - Make sure that the format of the tag you're pushing matches the pattern you've specified in the workflow configuration. The `**` pattern should match any tag.

2. **Workflow YAML File:**
   - Double-check that the `on` section of your workflow YAML file is correctly indented and structured. It should be under the `jobs` section and should look like this:

     ```yaml
     on:
       push:
         tags:
           - '**'
     ```

3. **Branch Protection:**
   - If you have branch protection rules set up for your repository, make sure that you have the required permissions to push tags. Sometimes, restrictions on tag pushes can prevent workflows from being triggered.

4. **Repository Settings:**
   - Go to your repository settings on GitHub and navigate to the "Actions" section. Make sure that GitHub Actions is enabled for your repository.

5. **Cached Workflow Runs:**
   - GitHub Actions caches workflow runs to prevent unnecessary executions. Check if there's a cached run of the workflow that might be causing the issue. You can clear the cache and then try pushing a new tag to trigger the workflow.

6. **GitHub Actions Logs:**
   - Check the GitHub Actions logs for any error messages or warnings related to the workflow. This can provide insights into why the workflow is not being triggered.

7. **Personal Access Token:**
   - If your repository is private and you're using a Personal Access Token (PAT) for authentication, ensure that the token has the required permissions to trigger workflows.

8. **GitHub Enterprise:**
   - If you're using GitHub Enterprise, make sure that the version of GitHub Enterprise you're using supports the features you're trying to use, including triggering workflows on tag pushes.

9. **Manual Workflow Runs:**
   - Try manually triggering the workflow using the "Run workflow" button on the Actions tab of your repository. This can help you determine if the issue is specific to automatic triggers.

If you've gone through these steps and the workflow is still not being triggered on tag pushes, it might be a good idea to reach out to GitHub Support for more detailed assistance. They can help you diagnose and resolve any specific issues related to your repository's configuration.

**Comment by nmccready** [comment]
Having the same problem and it is driving me nuts.
